soh/soh/Enhancements/randomizer/3drando/location_access/locacc_water_temple.cpp

```diff
@@ -40,7 +44,11 @@ void RegionTable_Init_WaterTemple() {
   areaTable[RR_WATER_TEMPLE_EAST_LOWER] = Region("Water Temple East Lower", "Water Temple", {RA_WATER_TEMPLE}, NO_DAY_NIGHT_CYCLE, {
                   //Events
                   EventAccess(&logic->CanWaterTempleLowFromHigh, {[]{return logic->CanWaterTempleLowFromHigh || logic->CanUse(RG_ZELDAS_LULLABY);}}),
-                }, {}, {
+                }, {
+                  //Locations
+                  LOCATION(RC_WATER_TEMPLE_TORCH_POT_1, logic->CanBreakPots()),
+                  LOCATION(RC_WATER_TEMPLE_TORCH_POT_2, logic->CanBreakPots()),
```

Comment:
These pots need guranteed water level low or irons + hook (I need to learn and clean up this mess of water logic at some point)